server/src/Controllers/ReleaseTestersController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using ReleaseMonkey.Server.Models;
+using ReleaseMonkey.Server.Services;
+
+namespace ReleaseMonkey.Server.Controller
+{
+
+    public record UpdateReleaseTesterRequest
+    (
+        int ReleaseTesterId,
+        int State,
+        string Comment
+    );
+
+    public record CreateReleaseTesterRequest
+    (
+        int ReleaseId,
+        int TesterId
+    );
+
+    [ApiController]
+    [Route("release-testers")]
+    public class ReleaseTestersController(ReleaseTestersService releaseTesters) : ControllerBase
+    {
+        [HttpGet]
+        public IActionResult Fetch()
+        {
+            return Ok(releaseTesters.GetAllReleaseTesters());
+        }
+
+        [HttpGet("{id:int}", Name = "FetchReleaseTesterById")]
+        public IActionResult FetchById(int id)
+        {
+            try
+            {
+                return Ok(releaseTesters.GetReleaseTesterById(id));
+            }
+            catch (KeyNotFoundException e)
+            {
+                return NotFound(e.Message);
+            }
+        }
+
+        [HttpGet("release/{id:int}", Name = "FetchReleaseTesterByReleaseId")]
+        public IActionResult FetchByProjectId(int id)
+        {
+            return Ok(releaseTesters.GetReleaseTestersByReleaseId(id));
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> Create(CreateReleaseTesterRequest body)
```

Comment:
Don't add endpoint to create ReleaseTester. They should be created automatically when we create a Release. The service for creating releases should automatically create them for the current testers.